core/deployment/src/main/java/io/quarkus/deployment/pkg/NativeConfig.java

```diff
@@ -48,6 +48,12 @@
     @ConfigItem(defaultValue = "false")
     public boolean addAllCharsets;
 
+    /**
+     * If all timezones sets should be added to the native image. This increases image size
```

Comment:
```suggestion
     * If all timezones sets should be added to the native image. This increases image size.
```

Reply:
We can live without exposing the config for now. So this will be removed with my coming commit.